src/main/java/org/folio/marccat/resources/domain/FixedField.java

```diff
@@ -3577,14 +3957,8 @@ public void setSequenceNumber(int sequenceNumber) {
   @Generated("org.jsonschema2pojo")
   public static enum MaterialType {
 
-    MUSIC("music"),
-    MAP("map"),
-    COMPUTER_FILE("computerFile"),
-    BOOK("book"),
-    MIXED_MATERIAL("mixedMaterial"),
-    CONTINUING_RESOURCE("continuingResource"),
-    VISUAL_MATERIAL("visualMaterial"),
-    UNDEFINED("undefined");
+    MUSIC("music"), MAP("map"), COMPUTER_FILE("computerFile"), BOOK("book"), MIXED_MATERIAL("mixedMaterial"),
+    CONTINUING_RESOURCE("continuingResource"), VISUAL_MATERIAL("visualMaterial"), UNDEFINED("undefined");
```

Comment:
Again, the old formatting is much easier to read IMO

Reply:
I have changed it